python/lsst/meas/algorithms/skyObjects.py

```diff
@@ -138,11 +138,27 @@ def generateSkyObjects(mask, seed, config):
 
 
 class SkyObjectsTask(Task):
-    """Generate a list of Footprints of sky objects.
+    """Generate a list of Footprints of sky sources (regions on the sky that
+    do not otherwise have detections).
+
+    Parameters
+    ----------
+    schema : `lsst.sfw.table.Schema`
```

Comment:
Typo, `sfw` to `afw`.

For parsed docstring purposes, this would read better as `~lsst.afw.table.Schema`.

Reply:
(Commenting post-merge, for reference only, so not expecting any changes to be made)

True, our dev guide doesn't say to shorten parameter types, but it also doesn't say not to either. This certainly crops up elsewhere in the stack, e.g. [daf_butler](https://github.com/lsst/daf_butler/blob/3e35ddf7b712e8d8088fef5484748109d5dd22a5/python/lsst/daf/butler/registry/interfaces/_datasets.py#L122) and also in other projects, e.g., [matplotlib](https://matplotlib.org/devdocs/devel/document.html#reference-types). I wouldn't mind having guidance on this in the dev guide, one way or another.

Reply:
This is worth asking about on docs-support: I could see it argued either way.